verl/workers/fsdp_workers.py

```diff
@@ -1455,10 +1429,19 @@ def execute_method(self, method: Union[str, bytes], *args, **kwargs):
             print(f"[DP={self.vllm_dp_rank},TP={self.vllm_tp_rank}] execute_method: {method if isinstance(method, str) else 'Callable'}")
         return self.rollout.execute_method(method, *args, **kwargs)
 
+    @register(dispatch_mode=Dispatch.DIRECT_ROLLOUT_METHOD, blocking=False)
```

Comment:
Is it possible to use the rollout API directly instead of adding a glue layer at the FSDP worker level?